auth_saml/models/base_settings.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2010-2016 XCG Consulting <http://odoo.consulting>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+_SAML_UID_AND_PASS_SETTING = 'auth_saml.allow_saml.uid_and_internal_password'
+
+
+class ResConfigSettings(models.TransientModel):
+    """Inherit from res.config.settings to add a setting. This is only here
+    for easier access; the setting is not actually stored by this (transient)
+    collection. Instead, it is kept in sync with the
+    "auth_saml.allow_saml.uid_and_internal_password" global setting. See
+    comments in the definition of the "res.config.settings" collection for
+    details.
+    """
+
```

Comment:
This is quite obvious, you don't need it:

```suggestion
```

Remember about docs:

1. Make code easy to read, so you don't have to document *what* you're doing.
1. Instead, document *why* when it isn't obvious.